cmake/modules/FindPortAudio.cmake

```diff
@@ -54,6 +54,11 @@ find_path(PortAudio_INCLUDE_DIR
   DOC "PortAudio include directory")
 mark_as_advanced(PortAudio_INCLUDE_DIR)
 
+find_path(PortAudio_ALSA_H
```

Comment:
Please add comments that this is only needed as a workaround, see below. We should remove `find_path(PortAudio_ALSA_H)` eventually.

Reply:
Unfortunately we will need it at least until our minimum supported OS packages the next release of PortAudio.

Reply:
also I'm not sure we'll be able to rely on PortAudio's build system while the maintainers continue to refuse to remove legacy build systems because I'm not duplicating implementing new build system features for them